### PR TITLE
Report: Display only first 100 rows when reporting table

### PR DESCRIPTION
**Pull request by VesnaT**
##### Includes
- [X] Code changes
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Orange/canvas/report/report.py [hunk 1]
```diff
@@ -291,7 +292,7 @@ def report_list(data,
             return join("  <tr>\n    {}</tr>\n".format(
                 join(cells[rowi < header_rows or coli < header_columns]
                      .format(fmtnum(elm)) for coli, elm in enumerate(row))
-            ) for rowi, row in enumerate(data))
+            ) for rowi, row in zip(range(row_limit + 1), data))
```
Comment (janezd) [inline Orange/canvas/report/report.py]: Doesn't this output `row_limit + 1` rows?

**Comment by janezd** [comment]
If the list is clipped, indicate this by adding a line (spanned across all columns)
`"+ {} more".format(len(...) - row_limit)`. Otherwise it gives an impression that there are exactly 100 items.

File: Orange/canvas/report/report.py (hunk 1 above)
Comment (VesnaT) [inline Orange/canvas/report/report.py]: No, I guess header is the first.

Comment (janezd) [inline Orange/canvas/report/report.py]: Isn't the number of header rows given as an argument, so it can also be 0 or 3?

**Pull request merged**